fighthealthinsurance/management/commands/generate_blog_metadata.py

```
@@ -100,7 +112,13 @@ def handle(self, *args, **options):
                     f"Wrote metadata for {len(posts)} posts to {output_path}"
                 )
             )
-        except Exception as e:
+        except OSError as e:
             self.stderr.write(
                 self.style.ERROR(f"Failed to write output file {output_path}: {e}")
             )
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)